staging/src/k8s.io/client-go/plugin/pkg/client/auth/gcp/gcp.go

```diff
@@ -300,6 +300,9 @@ func (c *commandTokenSource) Token() (*oauth2.Token, error) {
 	cmd.Stderr = &stderr
 	output, err := cmd.Output()
 	if err != nil {
+		if bytes.Contains(stderr.Bytes(), []byte("Reauthentication required")) {
+			return nil, fmt.Errorf("Reauthentication required. Please run:\n\n  gcloud auth login\n\nto obtain new credentials.")
```

Comment:
`gcloud auth login` will make me re-enter my password. I'd rather we pick a different command that just triggers the reauth flow.

Reply:
There is not a universal command to reauth. Some reauth flows even require external tools such as web browsers. So `gcloud auth login` is the safest fallback in this case, albeit more involved.

Reply:
I think the majority of usage for this provider is for OAuth clients, who can reauth without external tools, and I'd rather not train people to re-run `gcloud auth login` if they don't need to.

Reply:
As long as this is still depending on `gcloud` to obtain access tokens, `gcloud auth login` is the command to run. See associated bug for current error message. This CL doesn't change that fact.

Reply:
Sorry to go dark on this again. I would prefer to "fix" this in gcloud (e.g. give a better message when reauth is failing due to being called in a credential-helper). The change in this PR would definitely be helpful for a specific set of users, but I worry that it's too specific.